src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -39,66 +39,251 @@
 # are initially populated.
```

Comment:
Can you update this comment? I think we may still want a TODO to revert this back once we've decided on which temperature works best for our use case.